create_output_file.py

```diff
@@ -4,11 +4,19 @@
 
 output = []
 
-for file_object in os.scandir('releases'):
-  filename = file_object.name
-  if filename.endswith('.pdf'):
-    link = f'{base_url}{filename}'.replace(' ', '%20')
-    output.append({"name": filename.replace('.pdf', ''), "link": link})
-
-with open('index.json', 'w') as outputfile:
-  json.dump(output, outputfile)
+files = {'maintain': 'index.json', 'search': 'search.json'}
+
+def create_output(service):
+  output = []
+
+  for file_object in os.scandir(f'releases/{service}'):
+    filename = file_object.name
+    if filename.endswith('.pdf'):
+      link = f'{base_url}{service}/{filename}'.replace(' ', '%20')
```

Comment:
Probably want to use urllib.parse.quote or urlencode to do this in case someone put in some crazy characters

Reply:
Have done that now